src/main/template/main.handlebars

```diff
@@ -41,12 +41,6 @@
             </div>
 
             <p class="changes-disclaimer" style="display: none;">
-                <span class='info' id='api_info'>
-                    {{#if info}}
-                        <span class="info_title">{{info.title}}</span>
-                        <span class="markdown">{{{info.description}}}</span>
-                    {{/if}}
-
                 </span>
 
                 <span class='info' id='api_info'>
```

Comment:
I think this `info.contact.email` stuff should be removed from the template. Seems to already be hidden by CSS.